javascript/javascript1/week3/smartphone-usage.js

```diff
@@ -0,0 +1,42 @@
+const activities = [];
+function addActivity(date, activity, duration) {
+  activities.push({ date, activity, duration });
+  console.log(`${date} I watched ${activity} for ${duration} minutes`);
+}
+
+const limit = 100;
+
+let date = new Date().toLocaleDateString();
+
+console.log(activities);
+
+addActivity(date, "Youtube", 60);
+addActivity(date, "TikTok", 20);
+addActivity(date, "TV", 30);
+
+function showStatus() {
+  let activityTime = 0;
+  for (let i = 0; i < activities.length; i++) {
+    activityTime += activities[i].duration;
+  }
+
```

Comment:
you can take a look at `reduce` to get the total duration, that's a neater solution.